packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.ct.tsx

```diff
@@ -0,0 +1,47 @@
+import { expect, test } from "../../playwright-axe";
+import OnyxSwitch from "./OnyxSwitch.vue";
+
+test("should render default OnyxSwitch", async ({ mount, makeAxeBuilder }) => {
+  // ARRANGE
+  const component = await mount(<OnyxSwitch label="Switch" />);
+
+  // ASSERT
+  await expect(component).toContainText("Switch");
+  await expect(component).toHaveScreenshot("default.png");
+
+  // ACT
+  const accessibilityScanResults = await makeAxeBuilder().analyze();
+
+  // ASSERT
+  expect(accessibilityScanResults.violations).toEqual([]);
+});
+
+test("should render disabled OnyxSwitch", async ({ mount, makeAxeBuilder }) => {
+  // ARRANGE
+  const component = await mount(<OnyxSwitch label="Switch" disabled />);
+
+  // ASSERT
+  await expect(component).toContainText("Switch");
```

Comment:
```suggestion
  await expect(component).toContainText("Switch");
  await expect(component).toBeDisabled(); 
```

Reply:
I'll add this one, but I have prepared more tests for the second PR